core-bundle/src/Routing/ImagesLoader.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Routing;
+
+use Symfony\Component\Config\Loader\Loader;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Routing\Route;
+use Symfony\Component\Routing\RouteCollection;
+
+class ImagesLoader extends Loader
+{
+    /**
+     * @var string
+     */
+    private $pathPrefix;
+
+    public function __construct(string $projectDir, string $imageTargetDir, Filesystem $filesystem)
+    {
+        $this->pathPrefix = rtrim($filesystem->makePathRelative($imageTargetDir, $projectDir), '/');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function load($resource, $type = null): RouteCollection
+    {
+        $route = new Route('/'.$this->pathPrefix.'/{path<.+>}', ['_controller' => 'contao.controller.images']);
```

Comment:
Just making sure here: Can we use this short notation syntax already?

Reply:
Seems like we cannot ☺️. It’s a feature from Symfony 4.1. I will change.

Reply:
Fixed in 5cffe8ccbc430615616c87d11b8962f512e21446